### ISSUE TITLE
Add support for MacOs

### ISSUE THREAD

**Issue by daudsadiq1** [Redth/ResizetizerNT#38]

**Comment by icuric** [comment]
Hi,
any info on this? 
Is it true that project cannot build on MacOs?

I've just installed this plugin in Xamarin project but cannot build. I'm using Visual Studio for Mac.
Error msg says:
"Error: Unknown MSBuild failure. Please try building the project again"

**Comment by daudsadiq1** [comment]
Yes! ResizetizerNT does not support MacOS right now. Surprisingly it is available for WPF.

**Comment by icuric** [comment]
That is probably due to different build process on MacOS. 
Unfortunately that makes this tool unusable in real project as some devs are using VS for Mac...

I using workaround for now. Created new Xamarin forms project on Windows machine with only Resizetizer plugin installed, and in that project I'm creating images from .svg files and then manually copying images into main project.

Tool works great and is really helpful. I don't mind copying images manually as you do it only once, and it's is actually better like this because Resizetizer is not running on every build so build is not slowing down...

**Comment by daudsadiq1** [comment]
great!
